### PR TITLE
FIrst pull request

### PR DESCRIPTION
**Pull request by codezeeker**
Creating a pull request from the develop[a diffrent branch to the master branch].

- this allows different people to work on different features at the sametime without interfering each other!
- this also puts a approval/deny workflow - to allow or deny the change.

### CODE REVIEW

File: app/taskmanager.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
 cont = "y"
 lst = list()
 while cont == "y":
+    item = input("\nEnter the list item: ")
+    lst.append(item)
+    cont = input("\nDo you want to continue?[y/n] :")
 
-    lst = lst.append(input("enter the list item: "))
```
Comment (codezeeker) [inline app/taskmanager.py:5]: lst.append then assigning back to the lst caused issue. when i removed that it worked.

File: app/taskmanager.py [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
-    print("Here are the task list for today:", lst)
-    cont = input("Do you want to continue?[y/n] :")
-print(" -------------- ")
-print(" -------------- ")
-print("Have a good day")
\ No newline at end of file
+print("\n\n------------------")
+print("   Today's tasks")
+print("------------------")
+for i in lst:
```
Comment (codezeeker) [inline app/taskmanager.py:11]: just simple for loop helps you to iterate thru elements

Comment (codezeeker) [reply app/taskmanager.py:11]: @ahamed-sha  - Review the code and run to validate the program locally.

**Review by ghost — APPROVED** [review]
first review

**Pull request merged**